src/openrct2/ride/Ride.cpp

```diff
@@ -7211,10 +7211,10 @@ money16 ride_get_price(const Ride* ride)
  * Return the tile_element of an adjacent station at x,y,z(+-2).
  * Returns nullptr if no suitable tile_element is found.
  */
-TileElement* get_station_platform(int32_t x, int32_t y, int32_t z, int32_t z_tolerance)
+TileElement* get_station_platform(const CoordsXYRangedZ& coords)
```

Comment:
The z tolerance is different to a clearance z. A clearance z is meant to represent a height from the ground but z tolerance is the difference in height from the base to the clearance. We will need to modify the caller's and body to make it follow the expected conventions.

Reply:
If you are not quite sure what I mean its the following:

`z = 5, tolerance = 2, clearance = 7 = 5 + 2`

So all callers will need to be modified so that its passed `{ x, y, z, z + tolerance }`.
And the body will need to be modified so that it is just `clearnaceZ` instead of `z + tolerance`.

Reply:
Alright. Thank you!

Reply:
I updated it with what I understood. Also, if possible I need to reopen the PR.

Reply:
It looks like you deleted the branch on your fork and recreated it and that's why GitHub doesn't allow us to reopen it. I've never seen or done this, but I guess you'll have to open a new PR
![image](https://user-images.githubusercontent.com/6443427/87180083-39eea900-c2b6-11ea-8272-48e10231c28d.png)

Reply:
I understand you might have done this by accident. In the future a good tip is to try not to close and reopen PRs that are for the exact same thing, as we lose track of what had already been discussed

Reply:
Annoyingly if you delete your branch and then restore github will not let you reopen your PR. I've done this before and its very annoying.

Reply:
Second PR open and references this one. Thanks.